app_flutter/lib/widgets/commit_box.dart

```diff
@@ -111,26 +114,42 @@ class CommitOverlayContents extends StatelessWidget {
           top: offsetLeft.dy + (renderBox.size.height / 2),
           left: offsetLeft.dx + (renderBox.size.width / 2),
           child: Card(
-            child: Column(
-              mainAxisSize: MainAxisSize.max,
-              children: <Widget>[
-                ListTile(
-                  leading: CommitAuthorAvatar(commit: commit),
-                  // TODO(chillers): Show commit message here instead: https://github.com/flutter/cocoon/issues/435
-                  // Shorten the SHA as we only need first 7 digits to be able
-                  // to lookup the commit.
-                  title: SelectableText(commit.sha.substring(0, 7)),
-                  subtitle: SelectableText(commit.author),
-                ),
-                ButtonBar(
-                  children: <Widget>[
-                    ProgressButton(
-                      child: const Text('OPEN GITHUB'),
-                      onPressed: _openGithub,
+            child: SafeArea(
+              minimum: const EdgeInsets.all(16),
+              child: Row(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  CommitAuthorAvatar(commit: commit),
+                  Expanded(
+                    child: Padding(
+                      padding: const EdgeInsetsDirectional.only(start: 16),
+                      child: Column(
+                        crossAxisAlignment: CrossAxisAlignment.stretch,
+                        children: [
+                          AnimatedDefaultTextStyle(
+                            style: theme.textTheme.subtitle1,
+                            duration: kThemeChangeDuration,
+                            child: Hyperlink(
+                              text: commit.sha.substring(0, 7),
+                              onPressed: _openGithub,
+                            ),
+                          ),
+                          const SizedBox(height: 8),
+                          AnimatedDefaultTextStyle(
+                            style: theme.textTheme.bodyText2.copyWith(
+                              color: theme.textTheme.caption.color,
+                            ),
+                            duration: kThemeChangeDuration,
+                            child: SelectableText('${commit.message} ${commit.message} ${commit.message}'),
```

Comment:
three times?

Reply:
Oops - I was testing wrapping to see what it looked like.  Fixed.